Libs/DICOM/Core/ctkDICOMDisplayedFieldGeneratorStudyNumberOfSeriesRule.h

```diff
@@ -37,29 +37,29 @@ class CTK_DICOM_CORE_EXPORT ctkDICOMDisplayedFieldGeneratorStudyNumberOfSeriesRu
   explicit ctkDICOMDisplayedFieldGeneratorStudyNumberOfSeriesRule();
 
   /// Get name of rule
-  QString name()const override;
+  QString name()const CTK_OVERRIDE;
 
   /// Clone displayed field generator rule. Override to return a new instance of the rule sub-class
-  ctkDICOMDisplayedFieldGeneratorAbstractRule* clone() override;
+  ctkDICOMDisplayedFieldGeneratorAbstractRule* clone() CTK_OVERRIDE;
 
   /// Specify list of DICOM tags required by the rule. These tags will be included in the tag cache
-  QStringList getRequiredDICOMTags() override;
+  QStringList getRequiredDICOMTags() CTK_OVERRIDE;
 
   /// Generate displayed fields for a certain instance based on its cached tags
   /// The way these generated fields will be used is defined by \sa mergeDisplayedFieldsForInstance
   void getDisplayedFieldsForInstance(
     const QMap<QString, QString> &cachedTagsForInstance, QMap<QString, QString> &displayedFieldsForCurrentSeries,
-    QMap<QString, QString> &displayedFieldsForCurrentStudy, QMap<QString, QString> &displayedFieldsForCurrentPatient ) override;
+    QMap<QString, QString> &displayedFieldsForCurrentStudy, QMap<QString, QString> &displayedFieldsForCurrentPatient ) CTK_OVERRIDE;
 
   /// Start updating displayed fields (reset counters, etc.). No-op by default.
-  void startUpdate() override;
+  void startUpdate() CTK_OVERRIDE;
```

Comment:
I would consider reverting to custom code such as CTK_OVERRIDE, CTK_NULLPTR, Q_FOREACH,... is a step backward. It is worth doing it to have a clean state for Qt4/C++98 support. But once we are confident that it all works and created a tag/branch we should drop support for old stuff and get back to using modern API (go back to standard override, nullptr, etc.).